pkg/render/manager.go

```diff
@@ -516,9 +516,9 @@ func (c *managerComponent) voltronContainer() corev1.Container {
 // managerEsProxyContainer returns the ES proxy container
 func (c *managerComponent) managerEsProxyContainer() corev1.Container {
 	var keyPath, certPath string
-	if c.cfg.TLSKeyPair != nil {
+	if c.cfg.InternalTrafficSecret != nil {
```

Comment:
Please rename this to something like `InternalTLSKeyPair` instead of "TrafifcSecret" - the field is a `KeyPairInterface` and not a `corev1.Secret`